spec/metanorma/cli/generator_spec.rb

```diff
@@ -64,39 +69,39 @@
     context "with local template" do
       it "success for existing template" do
         document = "./tmp/my-local-document"
-        template = "./templates"
+        template = [Dir.home, ".metanorma", "templates", "csd"].join("/")
```

Comment:
This expects the template directory to contain the `CSD` template, which normally should be there since lots of other specs are cloning those templates before this command, but for some super random fact it could be the first specs to run and then it could have failed.

As an alternative to download it upfront (in each time a spec) runs, this might be a good approach to try it out and see if that actually happens.

Reply:
Good point